charts/rancher-gatekeeper/rancher-gatekeeper/100.0.1+up3.6.0/values.yaml

```diff
@@ -25,10 +25,13 @@ postInstall:
       pullSecrets: []
 image:
   repository: rancher/mirrored-openpolicyagent-gatekeeper
-  crdRepository: rancher/mirrored-openpolicyagent-gatekeeper-crds
   tag: v3.6.0
   pullPolicy: IfNotPresent
   pullSecrets: []
+crdImage:
+  repository: rancher/mirrored-openpolicyagent-gatekeeper-crds
+  tag: v3.6.0
+  pullPolicy: IfNotPresent
```

Comment:
Not a breaking change since this field never existed before:

https://github.com/rancher/charts/blob/5411430480180a486d2e067f46d0906b79938626/charts/rancher-gatekeeper/rancher-gatekeeper/100.0.0%2Bup3.5.1/values.yaml#L23-L27